tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -465,6 +465,7 @@ namespace Datadog.Trace
     public static class SpanTypes
     {
         public const string Benchmark = "benchmark";
+        public const string Browser = "browser";
```

Comment:
Can we just make this internal, I don't see why it needs to be public? 🤷‍♂️ + it will break the v3 branch snapshots too if you do 😛

Reply:
done in https://github.com/DataDog/dd-go/pull/132491/files